hw/dv/tools/dvsim/xcelium.hjson

```diff
@@ -35,6 +35,10 @@
                "-nowarn DSEMEL",
                // Ignore hierarchial ref warnings in interfaces
                "-nowarn CUVIHR",
+               // Needed for including "secded_enc.h".
+               "-I{build_dir}/src/lowrisc_dv_secded_enc_0",
```

Comment:
This is a "common" hjson file imported by all DUTs that may or may not need these switches. Ideal place to put this is in a dedicated hjson file. We can fix that later.